### PR TITLE
Control the collection view with the date scrubber

### PR DESCRIPTION
**Pull request by marijnschilling**
Still have some small weird "bugs" (not really bugs, just can't figure out the right settings)with bottom and top of the view when scrubbing 🙊

But I'll leave those for fresh ✨ tomorrow-morning-me (she can solve it in no time, for sure)

### CODE REVIEW

**Comment by 3lvis** [comment]
This is looking _pretty good_ 👏

File: Sources/DateScrubber.swift [hunk 1]
```diff
@@ -8,41 +8,88 @@
 
 import UIKit
 
+public protocol DateScrubberDelegate {
+     func requestToSetContentView(dateScrubber:DateScrubber, toYPostion yPosition: CGFloat)
```
Comment (3lvis) [inline Sources/DateScrubber.swift:12]: There's a typo here: `toYPostion`

Comment (marijnschilling) [inline Sources/DateScrubber.swift:12]: Cool! I'll fix the typo in the new branch i'm working on!

Comment (3lvis) [inline Sources/DateScrubber.swift:12]: okay ¯\_(ツ)_/¯

**Pull request merged**